src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = [...people];
+
+  century === undefined
+    ? men = men.filter(person => person.sex === 'm')
+    : men = men.filter(person => person.sex === 'm'
+      && Math.ceil(person.died / 100) === century);
```

Comment:
```suggestion
  const men = century === undefined
    ? people.filter(person => person.sex === 'm')
    : people.filter(person => person.sex === 'm'
      && Math.ceil(person.died / 100) === century);
```
Btw, you do filter in both cases, so you can just implement different logic inside filter callback